menu-bridges

```diff
@@ -138,72 +160,38 @@ read_config
 ###### DISPLAY THE MENU ######
 clear
 
-CHOICE=$(whiptail --cancel-button "Back" --title "TorBox v.0.4.0 - COUNTERMEASURE MENU" --menu "Choose an option (ESC -> back to the main menu)" $MENU_HEIGHT $MENU_WIDTH $MENU_LIST_HEIGHT \
+CHOICE=$(whiptail --cancel-button "Back" --title "TorBox v.0.4.0 - COUNTERMEASURE MENU" --menu "Choose an option (ESC -> back to the main menu)    ${BRIDGEMODE}${BRIDGESTRING}" $MENU_HEIGHT $MENU_WIDTH $MENU_LIST_HEIGHT \
 " " " " \
-"===" "============[Bypass Tor blocking with OBFS4 bridges -> "$TOGGLE05 \
-"  1" "ALL ABOUT OBFS4 BRIDGES - READ ME FIRST"  \
-"  2" "Toggle OBFS4 Bridge Mode from "$TOGGLE05b" to "$TOGGLE06b  \
-"  3" "List all "$number_configured_bridges_total" bridges"  \
-"  4" "Add additional bridges"  \
-"  5" "Activate configured bridges"  \
-"  6" "Deactivate configured bridges"  \
-"  7" "Remove configured bridges"  \
-"  8" "Enter the advanced configuration editor"  \
-"  9" "Restart Tor - check if it is working (press CTRL-C to leave)"  \
+"===" "============================================[Informational]===" \
+"  1" "ALL ABOUT BRIDGES - READ ME FIRST"  \
+"===" "=====================================[OBFS4 bridges -> "$TOGGLE05 \
+"  2" "List all "$number_configured_bridges_total" bridges"  \
+"  3" "Add additional bridges"  \
+"  4" "Activate configured bridges"  \
```

Comment:
OBFS4 should only be activated with the menu entry "Activate OBFS4 and configured bridges". Adding additional bridges shouldn't activate/deactivate the OBFS4 status.

Reply:
This is regarding the point below or there is another mentioned line in the script you can refer to me?